xml/installation-ardana-manila.xml

```diff
@@ -273,7 +539,7 @@ default_share_type = default1</screen>
      Create a &o_sharefs; share image and verify it
     </para>
 <screen>&prompt.ardana;wget http://tarballs.openstack.org/manila-image-elements/images/manila-service-image-master.qcow2
-&prompt.ardana;. service.osrc;glance image-create --name <literal>manila-service-image-new</literal> \
+&prompt.ardana;. service.osrc;openstack image create --name <literal>manila-service-image-new</literal> \
```

Comment:
`openstack image create <literal>manila-service-image-new</literal>`

Reply:
updated, same question as above... does this mean that the Cloud 9 docs need to be updated as well, or was this changed between Pike and Rocky?

Reply:
ditto